jobs/build/ocp4/Jenkinsfile

```diff
@@ -546,38 +397,43 @@ node {
             }
 
             stage("merge origin") {
+                // note, this will go away once merge is handled in its own job
                 dir(OSE_DIR) {
                     // Enable fake merge driver used in our .gitattributes
                     sh "git config merge.ours.driver true"
                     // Use fake merge driver on specific packages
                     sh "echo 'pkg/assets/bindata.go merge=ours' >> .gitattributes"
                     sh "echo 'pkg/assets/java/bindata.go merge=ours' >> .gitattributes"
 
-                    if (UPSTREAM_SOURCE_BRANCH != null) {
-                        // Merge upstream origin code into the ose branch
-                        sh "git merge -m 'Merge remote-tracking branch ${UPSTREAM_SOURCE_BRANCH}' ${UPSTREAM_SOURCE_BRANCH}"
-                    } else {
-                        echo "No origin upstream in this build"
+                    // Merge upstream origin code into the ose branch
+                    sh "git merge -m 'Merge remote-tracking branch ${UPSTREAM_SOURCE_BRANCH}' ${UPSTREAM_SOURCE_BRANCH}"
+                    if (!IS_TEST_MODE) {
+                        sh "git push"
                     }
+
                 }
             }
 
             stage("ose tag") {
                 dir(OSE_DIR) {
+                    // we no longer want to push tito's changes (and tags) to ose.
+                    // but push a tag for tito to generate the next changelog against and to match historical practice.
+                    def tag = "v${NEW_VERSION}-${NEW_RELEASE}"
+
+                    sh "git tag -am '' ${tag}"
+                    if (!IS_TEST_MODE) {
```

Comment:
Still concerned about pushing anything back to OSE, I do not believe we should be. Especially if we build the RPM with doozer.

Reply:
We have been pushing back. It doesn't look like we'll need to continue doing that. But this is not the final solution, this is reviving the ocp4 job for now.